core/src/main/java/org/geysermc/geyser/Constants.java

```diff
@@ -39,7 +39,7 @@ public final class Constants {
     public static final String GEYSER_DOWNLOAD_LOCATION = "https://geysermc.org/download";
     public static final String UPDATE_PERMISSION = "geyser.update";
 
-    static final String SAVED_REFRESH_TOKEN_FILE = "saved-refresh-tokens.json";
+    static final String SAVED_AUTH_CHAINS_FILE = "saved-auth-chains.json";
```

Comment:
If we're gonna move the file - which i'm personally not a huge fan of - can we atleast have some transitioning setup in place that would warn about people who have entries in the file? Maybe even (if possible) auto-migration, but at least something that would let them know that it is not further in use.

Reply:
Agreed. Can use Configurate

Reply:
Implemented auto-migration now, just needs to be tested (will do later)